include/client/audiomanager.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <unordered_map>
+#include <boost/filesystem.hpp>
+#include <SFML/Audio.hpp>
+#include <memory>
+
+enum class SoundType {
+    Collision,
+    Background
+};
+
+class AudioManager {
+public:
+	AudioManager();
+	~AudioManager();
+
+	void loadAudioFiles(std::vector< std::pair<boost::filesystem::path, SoundType>> audioPaths);
```

Comment:
One possible (nitpicky) change could be to make these load audio file functions private, and then have one public `init` function that loads all of the audio files we need by calling the private functions for all the audio files. It would make the AudioManager class less general, but since we are only using it for this project I think that would be okay, and it would clean up the client constructor / init function